pkg/signal/signal.go

```diff
@@ -30,7 +30,7 @@ type SignalType uint32
 const (
 	// SignalNatFillUp denotes potential congestion on the NAT table
 	SignalNatFillUp SignalType = iota
-	// SignalCTFillUp denotes potential congestion on the CT table
+	// SignalCTFillUp denotes potential congestion on the CT table. Unused.
```

Comment:
I'd maybe expand here, “Unused, given that the datapath no longer sends this signal”.